plugin/fcitx.vim

```diff
@@ -5,27 +5,66 @@ scriptencoding utf-8
 " URL:		https://www.vim.org/scripts/script.php?script_id=3764
 " ---------------------------------------------------------------------
 " Load Once:
-if &cp || exists("g:loaded_fcitx") || !exists('$DISPLAY') || !has('python3')
+if &cp || exists("g:loaded_fcitx") || !exists('$DISPLAY')
   finish
 endif
 let s:keepcpo = &cpo
 set cpo&vim
-let g:loaded_fcitx = 1
 
-try " abort on fail
-  exe 'py3file' expand('<sfile>:r') . '.py'
-  if py3eval('fcitx_loaded')
-    if exists('##InsertLeavePre')
-      au InsertLeavePre * py3 fcitx2en()
-    else
-      au InsertLeave * py3 fcitx2en()
+" If g:fcitx5_remote is set (to the path to `fcitx5-remove`), use it to toggle IME state.
+if exists("g:fcitx5_remote")
+  function Fcitx2en()
+    let inputstatus = system(g:fcitx5_remote)
+    if inputstatus == 2
```

Comment:
Shouldn't `inputstatus` be a string?

Reply:
This part of code is from branch `fcitx4`:
https://github.com/lilydjwg/fcitx.vim/blob/b6fc70ffba65f356e0a981dbec3b51cf471ea2c6/so/fcitx.vim#L23-L27

The `inputstatus` is actually like `2\n`. If compares as strings, maybe we need also `trim`.

Reply:
Oops. Yes, we need to trim and compare strings. Let's use the correct type to clear the way to rewrite it in Vim 9.